tsconfig.json

```diff
@@ -13,15 +12,18 @@
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "alwaysStrict": true,
```

Comment:
```suggestion
```
Our base config [uses `strict`](https://github.com/canonical/ds25/blob/dfa9c4e97cb5d24192c369e0359bf3559843c802/configs/typescript-base/tsconfig.json#L6), which includes `alwaysStrict` per [TS documentation](https://www.typescriptlang.org/tsconfig/#strict)

You should be able to remove `alwaysStrict`.